sdk/identity/identity/review/identity.api.md

```diff
@@ -134,7 +134,7 @@ export { GetTokenOptions }
 // @public
 export class InteractiveBrowserCredential implements TokenCredential {
     constructor(options?: InteractiveBrowserCredentialOptions);
-    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+    getToken(scopes: string | string[]): Promise<AccessToken | null>;
```

Comment:
Changes to this file mean changes to the public API of Identity. I do believe we want to avoid changing the public API of Identity as part of this PR, specially since the focus of this PR is the `tsconfig.json` file.

Reply:
Please correct me if I am wrong! I think `identity.api.md` is generated automatically when project is build. Since, we have removed the unused parameters it is reflected by this file. So if we exclude the `identity.api.md` changes will it not be out of sync with actual implementation?

Reply:
@mohsin-mehmood I believe the feedback here is that removing `options` from this method changes the API shape and thus would be considered as a breaking change. You observation is correct that `options` is unused in this method. We would need further investigation as to why `options` is unused. Please revert the changes in this file that removes the `options` parameter from the `getToken()` methods while we investigate this more

@sadasant, Can you log an issue to track this investigation as to why options is unused in the `getToken()` method for certain credentials?

Reply:
@sadasant and @ramya-rao-a With latest commit `options` parameter is restored

Reply:
Thank you Mohsin! I've made this issue for us to come back at this later: https://github.com/Azure/azure-sdk-for-js/issues/11482